src/macaron/slsa_analyzer/checks/provenance_available_check.py

```diff
@@ -56,11 +34,8 @@ class ProvenanceAvailableFacts(CheckFacts):
     #: The primary key.
     id: Mapped[int] = mapped_column(ForeignKey("_check_facts.id"), primary_key=True)  # noqa: A003
 
-    #: The provenance asset name.
-    asset_name: Mapped[str] = mapped_column(String, nullable=True, info={"justification": JustificationType.TEXT})
-
-    #: The URL for the provenance asset.
-    asset_url: Mapped[str] = mapped_column(String, nullable=True, info={"justification": JustificationType.HREF})
+    #: The availability of the provenance.
+    available: Mapped[str] = mapped_column(String, nullable=True, info={"justification": JustificationType.TEXT})
```

Comment:
I don't think this column is adding any valuable info. By the check result, we can already know whether a provenance is available or not and this column can be removed.